.golangci.yml

```diff
@@ -73,6 +73,7 @@ linters-settings:
       - name: error-strings
       - name: error-return
       - name: receiver-naming
+      - name: unused-parameter
```

Comment:
Hi, @Affan-7 , To avoid multiple PR conflicts, you can add this and fix it locally. The modifications here will be submitted together in the end.

Reply:
Never mind, this is the last linting task assigned to @Affan-7.